auslib/web/admin/views/rules.py

```diff
@@ -117,10 +118,13 @@ def _post(self, id_or_alias, transaction, changed_by):
 
         what, mapping_values, fallback_mapping_values = process_rule_form(connexion.request.get_json())
 
-        if what.get('mapping', None) is not None and len(mapping_values) != 1:
+        if 'mapping' in what and not what.get('mapping', None):
+            return problem(400, 'Bad Request', 'mapping value cannot be set to null/empty')
+
+        elif what.get('mapping', None) and len(mapping_values) != 1:
             return problem(400, 'Bad Request', 'Invalid mapping value. No release name found in DB')
 
-        if what.get('fallbackMapping', None) is not None and len(fallback_mapping_values) != 1:
+        elif what.get('fallbackMapping') is not None and len(fallback_mapping_values) != 1:
```

Comment:
reason being that empty string "" can be passed in the input and comparing truthfulness is to be a better option of handling both None and '' together. ( int(0) or bool(False) will never be passed for a string field 'mapping'). Hence this check : what.get('mapping') covers all.

When 

> 'mapping' in what and not what.get('mapping', None) 

condition is present , it implies that if mapping was present in request.json and if has a Null/Blank value then execute the following statement.

Whereas:
> not what.get('mapping', None) 
statement checks has 2 checks in 1:  (a) if mapping was passed in request.json, then check if it's value was blank/empty . (b) if mapping wasn't passed at all in request.json then perform  following statement . This condition is only used in "change_type" == "insert" cases.

fallbackMapping can be null/blank but mapping cannot be.